src/modules/rlm_eap/types/rlm_eap_fast/rlm_eap_fast.c

```diff
@@ -331,10 +331,10 @@ static int _session_ticket(SSL *s, uint8_t const *data, int len, void *arg)
 	     vp = fr_pair_list_next(&fast_vps, vp)) {
 		if (vp->da == attr_eap_fast_pac_info_pac_type) {
 			fr_assert(t->pac.type == 0);
-			t->pac.type = vp->vp_uint32;
+			t->pac.type = vp->vp_time_delta;
 		} else if (vp->da == attr_eap_fast_pac_info_pac_lifetime) {
 			fr_assert(t->pac.expires == 0);
-			t->pac.expires = request->packet->timestamp + fr_time_delta_from_sec(vp->vp_uint32);
+			t->pac.expires = request->packet->timestamp + fr_time_delta_from_sec(vp->vp_time_delta);
```

Comment:
`vp->vp_time_delta` is already `fr_time_t`.  You don't need to convert it to time_delta.